doc/nrf/libraries/modem/modem_slm.rst

```diff
@@ -7,7 +7,7 @@ Modem SLM
    :local:
    :depth: 2
 
-The Modem SLM library exposes the AT command interface of the Serial LTE Modem (:ref:`slm_description`) for external devices over a serial interface.
+The Modem SLM library exposes the AT command interface of the Serial LTE Modem application (:ref:`slm_description`) for external devices over a serial interface.
```

Comment:
```suggestion
The Modem SLM library exposes the AT command interface of the :ref:`Serial LTE Modem <slm_description>` application for external devices over a serial interface.
```
Since we're editing the line, might as well make the link according to our current style.